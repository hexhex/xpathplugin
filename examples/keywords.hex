% select all keywords from scientists.xml

keyword(X) :- &xpath["scientists.xml","//keyword/child::node()"](X).