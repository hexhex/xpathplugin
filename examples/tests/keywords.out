{keyword("Radioactivity"), keyword("Polonium"), keyword("Radium"), keyword("iWoz"), keyword("Macintosh"), keyword("First Personal Computer"), keyword("Apple"), keyword("Das seelenleben des Jugendlichen"), keyword("Humanistic Psychology"), keyword("Vienna Definition Language"), keyword("URRI"), keyword("IBM"), keyword("Mailuefterl"), keyword("Nobel Price"), keyword("quantum theory"), keyword("special thory of relativity"), keyword("theory of relativity")}
